package.json

```diff
@@ -22,6 +22,109 @@
   ],
   "main": "./out/extension/extension.js",
   "contributes": {
+    "configuration": [
+      {
+      "title": "Solc compiler defaults",
+      "order": 0,
+      "properties": {
+        "CompilerFolder": {
+          "type": "string",
+          "description": "Folder containing solc compiler binary file.\nNeed to supply if not part of path.",
```

Comment:
Nurit said to not use the word "path" yesterday in the meeting because it was confusing...

Reply:
Oh, I meant to say it should be `...to supply if not part of the PATH.` (instead of `...part of path`).

(It's the `PATH` variable. It's not just any path...)